app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -65,27 +72,29 @@ export const ScheduleDayPicker: FC<ScheduleDayPickerProps> = ({
     isSameDay(date, selectedScheduleDate),
   )
   const containerRef = React.useRef<View>()
-  const [measures, setMeasures] = React.useState([{ x: 0 }, { x: 0 }, { x: 0 }])
   const itemRefs = scheduleDates.map((_) => React.createRef<View>())
+  const [measures, setMeasures] = React.useState<Measurement[]>(
+    itemRefs.map((_) => {
+      return { x: 0, y: 0, width: 0, height: 0 }
```

Comment:
extract "default" value into typed const above so you can use here and line 91

Reply:
Addressed in [f54665e](https://github.com/infinitered/ChainReactApp2023/pull/212/commits/f54665e7affa49922804839adc04961b4f1bf9d2)